website/client/src/components/groups/questSidebarSection.vue

```diff
@@ -149,7 +149,19 @@
             <div class="row boss-details">
               <div class="col-6">
                 <span class="float-left">
-                  {{ Math.ceil(parseFloat(group.quest.progress.hp) * 100) / 100 }} / {{ parseFloat(questData.boss.hp).toFixed(2) }} <!-- eslint-disable-line max-len -->
+                  {{
+                    (Math.ceil(parseFloat(group.quest.progress.hp) * 100) / 100)
+                      .toLocaleString(user.preferences.language, {
+                        minimumFractionDigits: 2,
+                        maximumFractionDigits: 2
+                      })
+                  }} / {{
+                    parseFloat(questData.boss.hp)
+                      .toLocaleString(user.preferences.language, {
+                        minimumFractionDigits: 2,
+                        maximumFractionDigits: 2
+                      })
```

Comment:
Would be good to externalize this functionality into a method.  Perhaps something like `localizeNumber(number, digits)`

Reply:
Okeydokey, I was wondering if I should do this.